code/modules/clothing/masks/gasmask.dm

```diff
@@ -78,7 +80,10 @@
 
 /// Initializes the FoV component for the gas mask
 /obj/item/clothing/mask/gas/proc/init_fov()
-	AddComponent(/datum/component/clothing_fov_visor, FOV_90_DEGREES)
+	if (has_fov)
+		AddComponent(/datum/component/clothing_fov_visor, FOV_90_DEGREES)
+	else
```

Comment:
is this part necessary?

Reply:
yes, the alternative is manually overriding each `init_fov()` proc like it does now (see https://github.com/tgstation/tgstation/pull/63535)

Reply:
oh wait you mean the `else return` part don't you, yeah ishould remove that

Reply:
overriding `init_fov()` is totally fine though

Reply:
yeah but wouldnt it make more sense to do the above, instead of overriding it for every mask that doesn't have an fov

Reply:
Redirect these to #63535.

Reply:
huh? `else return` is still in there

Reply:
See https://github.com/tgstation/tgstation/pull/63535/commits/61c4f07ef0e4cc21600c937eb5db3b08e48e8aad, I simply didn't pull that commit yet.

Reply:
ah